docs/products/kafka/howto/configure-with-kafka-cli.rst

```diff
@@ -0,0 +1,53 @@
+Manage configurations with Apache Kafka® CLI tools
+==================================================
+
+ZooKeeper is an important part of Apache Kafka® since it tracks the status of the cluster and offers visibility its the current state. Moreover, by interacting with ZooKeeper, you're able to change any Apache Kafka parameter.
+
+Aiven for Apache Kafka® is a fully managed service, therefore, to guarantee the service stability, good maintenance and functionality, we do not allow direct access to ZooKeeper. Instead, we monitor and manage it using the Aiven platform to make sure that we can react as quickly as possible if any issues occur.
+
+Most of the changes usually performed via ZooKeeper are available via the :doc:`advanced configuration flags in the Aiven web console <use-zookeeper>` or via the :doc:`Aiven CLI </docs/tools/cli>`. However, you may want to use the client tools included in the `Apache Kafka binaries <https://kafka.apache.org/downloads>`_ to manage the configurations for your services. 
+
+.. Warning::
+    
+    Without access to ZooKeeper, some of the Apache Kafka command line tools are not working. But some commands, with the Apache Kafka product maturation, removed the reliance on ZooKeeper giving you options to handle specific configuration changes.
+
+Example: Create a topic with retention time to 30 minutes with ``kafka-topics.sh``
+----------------------------------------------------------------------------------
+
+Each topic in Apache Kafka can have a different retention time, defining for the messages' time to live. The Aiven web console and API offer the ability to set the retention time as part of the topic :ref:`creation <avn_cli_service_topic_create>` or :ref:`update <avn-cli-topic-update>`.
```

Comment:
Is there an example of this configuration in the CLI docs?

Reply:
Added the example. The example needs to be reworked once we enable the new config in Aiven client (I will check the dedicated code)